tests/testthat/helper-envvars-and-paths.R

```diff
@@ -10,8 +10,14 @@ not_on_cran <- function() {
   on_ci() || identical(Sys.getenv("NOT_CRAN"), "true")
 }
 
-test_release_url <- function() {
-  "https://github.com/stan-dev/cmdstan/releases/download/v2.25.0/cmdstan-2.25.0.tar.gz"
+mpi_toolchain_present <- function() {
```

Comment:
This is used instead of CMDSTANR_RUN_MPI_TESTS. If there are mpicxx and mpiexec present we run tests, otherwise not.